lumicks/pylake/force_calibration/power_spectrum_calibration.py

```diff
@@ -133,16 +131,18 @@ def guess_f_diode_initial_value(ps, guess_fc, guess_D):
     """
     f_nyquist = ps.sample_rate / 2
     P_aliased_nyq = (guess_D / (2 * math.pi ** 2)) / (f_nyquist ** 2 + guess_fc ** 2)
-    if ps.P[-1] < P_aliased_nyq:
-        dif = ps.P[-1] / P_aliased_nyq
+    if ps.power[-1] < P_aliased_nyq:
+        dif = ps.power[-1] / P_aliased_nyq
         return math.sqrt(dif * f_nyquist ** 2 / (1.0 - dif))
     else:
         return 2 * f_nyquist
 
 
-def calculate_power_spectrum(data, sample_rate, fit_range=(1e2, 23e3), num_points_per_block=350,
-                             compatibility_mode=False):
-    """Compute power spectrum.
+def calculate_power_spectrum(
+    data, sample_rate, fit_range=(1e2, 23e3), num_points_per_block=350, compatibility_mode=False
+):
+    """Compute power spectrum and returns it as a
+    `lumicks.pylake.force_calibration.power_spectrum.PowerSpectrum`.
```

Comment:
Use Sphinx notation to make sure that this becomes a link in the docs. I'm pretty sure that you also don't need to make it this long: 
```
:class:`~.PowerSpectrum`
```
That should work. The `.` tell it to look for the name in the current module and `~` tells it to also use this short name for the rendered docs (instead of the usual fully qualified name). See the sphinx docs and experiment with this a bit. It would be good to have all this markup applied consistently in the docs for a good user experience: https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-syntax

Reply:
Thanks. I updated the docs for force calibration in this fashion and will keep a lookout in the future.